### PR TITLE
FIPS image: also set OPENSSL_FIPS=1 and update README

### PR DESCRIPTION
**Pull request by dirkmueller**

### CODE REVIEW

File: src/bci_build/package/base-fips/README.md.j2 [hunk 1]
```diff
@@ -2,23 +2,37 @@
...
-environment variable `OPENSSL_FORCE_FIPS_MODE`. This variable is set to `1` in
-the image. This means that all cryptographic operations performed by the
-OpenSSL library are done in FIPS-140-2 mode.
+The image is configured to enforce the use of FIPS-140 mode by default independent
+of the host environment setting via the following configured environment variables:
+* `OPENSSL_FIPS=1`: Initialize the OpenSSL FIPS-140 mode
+* `OPENSSL_FORCE_FIPS_MODE=1`: Set FIPS mode to enforcing independent of the host kernel
+* `LIBGCRYPT_FORCE_FIPS_MODE=1`: Set FIPS mode in libgcrypt to enforcing
+
+The following environment variables environment variables are available to configure
```
Comment (dcermak) [inline src/bci_build/package/base-fips/README.md.j2:30]:
```suggestion
The following environment variables are available to configure
```

File: src/bci_build/package/base-fips/README.md.j2 [hunk 2]
```diff
@@ -2,23 +2,37 @@
...
+further aspects of the OpenSSL library:
+
+* `OPENSSL_ENFORCE_MODULUS_BITS=1`: Restrict the OpenSSL module to only generate
+the acceptable key sizes of RSA.
```
Comment (dcermak) [inline src/bci_build/package/base-fips/README.md.j2:34]: Can you please add a link to further information about this or is this common knowledge in the FIPS world?
```suggestion
* `OPENSSL_ENFORCE_MODULUS_BITS=1`: Restrict the OpenSSL module to only generate
the acceptable key sizes of RSA.
```

Comment (dirkmueller) [reply src/bci_build/package/base-fips/README.md.j2:34]: @dcermak I wish I would have that link

File: src/bci_build/package/base-fips/README.md.j2 [hunk 3]
```diff
@@ -2,23 +2,37 @@
 {% include 'badges.j2' %}
 
 ## Description
-This container image is a SUSE Linux Enterprise 15 SP3 base container image
-that includes the SLES 15 FIPS-140-2 certified OpenSSL module.
+This SUSE Linux Enterprise 15 SP3 based container image includes the
```
Comment (dmpop) [inline src/bci_build/package/base-fips/README.md.j2:5]:
```suggestion
This SUSE Linux Enterprise 15 SP3-based container image includes the
```

File: src/bci_build/package/base-fips/README.md.j2 [hunk 4]
```diff
@@ -2,23 +2,37 @@
...
+same functionality as the standard libgcrypt library, with the additional
+security features enforced to meet FIPS-140-2 requirements.
+
 ## Usage
-The image is configured to enforce the use of FIPS-140 mode by default via the
-environment variable `OPENSSL_FORCE_FIPS_MODE`. This variable is set to `1` in
-the image. This means that all cryptographic operations performed by the
-OpenSSL library are done in FIPS-140-2 mode.
+The image is configured to enforce the use of FIPS-140 mode by default independent
+of the host environment setting via the following configured environment variables:
```
Comment (dmpop) [inline src/bci_build/package/base-fips/README.md.j2:25]:
```suggestion
The image is configured to enforce the use of FIPS-140 mode by default, independent of the host environment setup. This is achieved by specifying the following environment variables:
```

File: src/bci_build/package/base-fips/README.md.j2 [hunk 5]
```diff
@@ -2,23 +2,37 @@
...
+* `OPENSSL_FIPS=1`: Initialize the OpenSSL FIPS-140 mode
+* `OPENSSL_FORCE_FIPS_MODE=1`: Set FIPS mode to enforcing independent of the host kernel
+* `LIBGCRYPT_FORCE_FIPS_MODE=1`: Set FIPS mode in libgcrypt to enforcing
+
+The following environment variables are available to configure
+further aspects of the OpenSSL library:
```
Comment (dmpop) [inline src/bci_build/package/base-fips/README.md.j2:31]:
```suggestion
Below is a list of other environment variables that can be used to configure the OpenSSL library:
```

**Comment by msmeissn** [comment]
OPENSSL_FIPS is for the commandline tool . it might make sense to have it, yes. so its ok

**Review by msmeissn — COMMENTED** [review]
ok

**Pull request merged**